continuous-upgrade/jobs/install-ocp.sh

```diff
@@ -0,0 +1,37 @@
+script="$( mktemp )"
+cat <<SCRIPT >"${script}"
```

Comment:
We had to do this because we were not using JJB -- I think we should be good to just use a script block in the YAML